R/aaa-auto.R

```diff
@@ -2080,6 +2080,38 @@ is_bipartite_impl <- function(graph) {
   res
 }
 
+
+bipartite_game_gnp_impl <- function(n1, n2, p, directed=FALSE, mode=c("all", "out", "in", "total")) {
+  # Argument checks
+  n1 <- as.numeric(n1)
+  n2 <- as.numeric(n2)
+  p <- as.numeric(p)
+  directed <- as.logical(directed)
+  mode <- switch(igraph.match.arg(mode), "out"=1L, "in"=2L, "all"=3L, "total"=3L)
+
+  on.exit( .Call(R_igraph_finalizer) )
+  # Function call
+  res <- .Call(R_igraph_bipartite_game_gnp, n1, n2, p, directed, mode)
+  res <- set_vertex_attr(res$graph, "type", value = res$types)
```

Comment:
@Antonov548 I mean this line in this and the other function. Furthermore there should be NO

```r
  if (igraph_opt("add.vertex.names") && is_named(graph)) {
    names(res) <- vertex_attr(graph, "name")
  }
  
```

Thanks!

Reply:
Hello @maelle 

I just took a look how to update generation to match with existing code. It is just first time we setting vertex attribute in generated interface.

I think I understand why `names(res$types) <- vertex_attr(graph, "name", V(graph))` is not needed, but do we really need to `set_vertex_attr`? I just trying to understand it in order to decide how to update stimulus part.

Reply:
Hi @Antonov548 maybe we could add `set_vertex_attr()` in the not automatic code. Let me discuss this with @krlmlr and @schochastics and then I'll update this thread (within the next few weeks). Thank you.

Reply:
> maybe we could add `set_vertex_attr()` in the not automatic code.

Good. To be honest I think not automatic code maybe it better place for setting up the attributes. I will wait for the decision.

Reply:
If we change in the template, how many functions will be affected?

@Antonov548: Can you please adapt the template and run the generation? We can revert later if needed.